project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/components/list/images.less

```diff
@@ -1,48 +1,54 @@
 @list-images-gap: 20px;
 @list-images-height: 100px;
-@list-images-label-height: 30px;
+@list-images-label-line-height: 16px;
+@list-images-label-max-rows: 2;
 
 .list-images {
     .reset-ul();
     margin-left: -@list-images-gap;
     margin-bottom: @margin-gap;
     display: flex;
+    flex-direction: row;
     flex-wrap: wrap;
 
     &__item {
-        flex: 0 100%/2;
+        display: flex;
+        flex-direction: column;
+        width: 50%;
         padding-left: @list-images-gap;
         margin-bottom: @list-images-gap;
 
         @media @query-sm {
-            flex: 0 100%/3;
+            width: 100%/3;
```

Comment:
Should not we use flex basis instead of width?

Reply:
This article clearly describes advantages of using flex-basis. 
https://www.freecodecamp.org/news/flexboxs-flex-basis-explained-83d1a01413b7/

But in this case we want to use only calculated width.

Reply:
Now I see. So it is fine for me. :)